corehq/apps/hqwebapp/static/cloudcare/scss/formplayer-webapp/menu.scss

```diff
@@ -23,21 +27,24 @@
 }
 
 #persistent-menu-container {
-  padding: 1.5rem 1.5rem 2.5rem 2.5rem;
+  padding: 1.5rem;
   font-size: 14px;
+  overflow-x: visible;
+  overflow-y: scroll;
+  z-index: $zindex-persistent-menu-nav;
 }
 
 #persistent-menu-container .btn-close {
   right: 15px;
   top: 15px;
 }
 
-.persistent-menu-toggle {
-  color: white;
-  background-color: $cc-brand-mid;
-  padding: $spacer / 2;
+#persistent-menu-arrow-toggle {
+  color: black;
+  z-index: $zindex-persistent-menu-nav + 1;
```

Comment:
nit: is there a way to define this directly in the variables files? like `@zindex-persistent-menu-nav-arrow = $zindex-persistent-menu-nav + 1`? Just to make it easier to find if anyone works in that area.